server/pulp/server/webservices/views/repositories.py

```diff
@@ -79,45 +81,43 @@ def _convert_repo_dates_to_strings(repo):
         repo['last_unit_removed'] = dateutils.format_iso8601_datetime(new_date)
 
 
-class ReposView(View):
-    """
-    View for all repos.
+def _process_repos(repos, details, importers, distributors):
     """
+    Apply standard processing to a collection of repositories being returned to a client. Adds
+    the object link and optionally adds related importers and distributors.
 
-    @staticmethod
-    def _process_repos(repos, importers=False, distributors=False):
-        """
-        Apply standard processing to a collection of repositories being returned to a client. Adds
-        the object link and optionally adds related importers and distributors.
-
-        :param repos: collection of repositories
-        :type  repos: list, tuple
-        :param importers: if True, adds related importers under the attribute "importers".
-        :type  importers: bool
-        :param distributors: if True, adds related distributors under the attribute "distributors"
-        :type  distributors: bool
+    :param repos: collection of repositories
+    :type  repos: list, tuple
+    :param importers: if `true`, adds related importers under the attribute "importers".
+    :type  importers: str
+    :param distributors: if `true`, adds related distributors under the attribute "distributors"
+    :type  distributors: str
 
-        :return: the same list that was passed in, just for convenience. The list itself is not
-                 modified- only its members are modified in-place.
-        :rtype:  list of Repo instances
-        """
-
-        if importers:
-            _merge_related_objects(
-                'importers', manager_factory.repo_importer_manager(), repos)
-        if distributors:
-            _merge_related_objects(
-                'distributors', manager_factory.repo_distributor_manager(), repos)
-
-        for repo in repos:
-            repo['_href'] = reverse('repo_resource', kwargs={'repo_id': repo['id']})
-            _convert_repo_dates_to_strings(repo)
+    :return: the same list that was passed in, just for convenience. The list itself is not
+             modified- only its members are modified in-place.
```

Comment:
This also might be confusing to readers. Might be best to just not return the list so that it's very clear what this is doing.